src/js/hooks/cycleCount/useAllProductsTab.jsx

```diff
@@ -20,15 +23,20 @@ const useAllProductsTab = () => {
     currentLocale: state.session.activeLanguage,
   }));
 
-  const { tableData, loading } = useTableData({
+  const {
+    tableData,
+    loading,
+  } = useTableData({
     filterParams: {},
     // Should be replaced after integrating with backend
     url: 'cycleCount',
     errorMessageId: 'react.cycleCount.table.errorMessage.label',
     defaultErrorMessage: 'Unable to fetch products',
     fetchManually: true,
-    getParams: () => {},
-    onFetchedData: () => {},
+    getParams: () => {
+    },
+    onFetchedData: () => {
+    },
```

Comment:
if you are using prettier, please turn it off, because the autoformatting will make the review harder (a lot of changes that are doing nothing)